week-2-practise/level-2/4-slice.js

```diff
@@ -5,7 +5,8 @@
 // - remove the item at the specified index
 
 function remove(arr, index) {
-  return; // complete this statement
+  arr.splice(2, 1);
```

Comment:
Hmm, the function should work for any array and index. How can we use the parameters instead?